packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -746,6 +758,13 @@ export const Audio = () => {
     handlePlaybackRateChange()
   }, [handlePlaybackRateChange, playbackRate])
 
+  useEffect(() => {
+    // Stop playback if we have unloaded a uid from the player
+    if (previousUid && !uid && !playing) {
+      handleStop()
+    }
+  }, [handleStop, playing, uid, previousUid])
```

Comment:
Also YIL ['you may not need a hook'](https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state) for this...